.travis.yml

```diff
@@ -164,17 +165,17 @@ script:
   - python -m pytest -v --durations=10 python/ray/tune/test/automl_searcher_test.py
 
   # ray rllib tests
-  - python -m pytest -v --durations=10 python/ray/rllib/test/test_catalog.py
-  - python -m pytest -v --durations=10 python/ray/rllib/test/test_filters.py
-  - python -m pytest -v --durations=10 python/ray/rllib/test/test_optimizers.py
-  - python -m pytest -v --durations=10 python/ray/rllib/test/test_evaluators.py
+  - python python/ray/rllib/test/test_catalog.py
```

Comment:
The --pyargs ray argument doesn't work here (it would try to execute all tests, but some are to be executed only in jenkins); this is temporary anyways, so I think it's fine (see https://github.com/ray-project/ray/issues/4031 for context)